docs/guides/installation_uv.md

```diff
@@ -0,0 +1,260 @@
+
+```bash
+source .venv/bin/activate
+marimo edit hi.py
+``` 
+
+---
+### Using Specific Python Versions
+
+To specify a Python version, add the `--python` option while creating the venv. For example:
+
+```bash
+uv venv --python 3.13
+```
+
+```{admonition} Tip
+:class: tip
+
+uv will automatically fetch Python versions as needed — you don't need to install Python to get started.
+```
+<!-- Source: https://docs.astral.sh/uv/guides/install-python/#installing-python -->
+
+
+If no version is specified, the default Python version is used (currently: **[TODO: Specify default Python version]**).
+
+---
+
+### Installing Packages
+
+There are three ways to install additional packages:
+
+1. **Via Terminal**  
+   Run the following command:  
+   ```bash
+   uv pip install matplotlib polars
+   ```
+
+2. **Within the Notebook**  
+   Type `import polars as pd`. Run the cell, and a "Missing package" pop up window will appear.
+
+<div align="center">
+<figure>
+<img src="/_static/image.png" width="650px"/>
+<figcaption>
+Screenshot demonstrating the "Missing package" window.
+</figcaption>
+</figure>
+</div>
+
+
+3. **Using the Packages Tab**  
+   Navigate to the "Packages" tab and select the desired package.
+
+<div align="center">
+<figure>
+<img src="/_static/image-1.png" width="400px"/>
+<figcaption>
+Screenshot demonstrating the "Packages" tab.
+</figcaption>
+</figure>
+</div>
+
+
+### Defining Dependencies with `pyproject.toml`
+
+A pyproject.toml file makes it easier to manage your project’s dependencies in one place. Using `uv`, you can quickly set up and customize your project’s environment
+
+```bash
+uv init           # Creates a pyproject.toml file
+uv add marimo     # Adds "marimo>=0.9.31" to dependencies
+uv run marimo edit hi.py
+```
+
+
+### Reading a `pyproject.toml` from an Existing Project
+
+If you already have a `pyproject.toml` file—for example, when cloning an existing project—you can use the `uv sync` command to synchronize and install the dependencies defined within it:
+
+```bash
+uv sync
+uv run marimo edit hi.py
+
+```
+
+
+
+This command ensures that your environment matches the dependency specifications of the existing project, making it simple to get up and running without manually adding packages.    
+
+
+## Temporary Installation
+
+When you execute a command using `uv tool run`, no virtual environment folder is created in your working directory. Instead, `uv` takes the following steps:
+
+1. **Caches dependencies or reuses existing cached ones** based on the metadata in the Marimo notebook.
+2. **Creates a temporary virtual environment** on your system.
+3. **Deletes the temporary environment** immediately after the process ends.
+
+This lightweight approach keeps your workspace clean while still providing an isolated, dependency-managed environment for running commands.
+
+For instance, you can run:
+
+```bash
+uv tool run marimo edit hi.py
```

Comment:
we should just always suggest using `--sandbox` when using `uv tool run`, otherwise they lose their package metadata

Reply:
Good point! Maybe `--sandbox` can even become be the default when using `uv tool run`?